_content/about.md

```diff
@@ -1 +1,5 @@
+---
+layout: main.njk
+---
```

Comment:
Since our top level config file isn't currently working, I had to explicitly specify our default `layout` in the top level templates.